news/2016-10-07-tip2-devmode.asciidoc

```diff
@@ -0,0 +1,126 @@
+---
+layout: news
+title: Weld Tip 2 - Development Mode
+author: Martin Kouba
+priority: 1
+change_frequency: weekly
+excerpt: Meet Weld Probe - a development tool which allows to inspect CDI components at runtime.
+desc: Weld Tip 2 - Development Mode
+tags: [tips]
+---
+:linkattrs:
+:toc: macro
+:toc-title: Contents
+
+toc::[]
+
+In this article we're going to cover the link:http://docs.jboss.org/weld/reference/latest/en-US/html/devmode.html[special mode for application development, window="_blank"].
+When enabled, a built-in tool called *Probe* provides a detailed view into internals of a CDI application.
```

Comment:
Actually, I've borrowed this from Jozef's [blogpost](http://weld.cdi-spec.org/news/2015/02/05/weld-300Alpha4/) ;-)
`detailed insight into` looks better.